src/build.props

```diff
@@ -24,8 +24,8 @@
       NOTE: Version in scripts/BuildMultitoolForNpm.ps1 also has to be updated for each release
       until it adopts the same versions as the SDK itself.
     -->
-    <VersionPrefix>2.2.0</VersionPrefix>
-    <PreviousVersionPrefix>2.1.25</PreviousVersionPrefix>
+    <VersionPrefix>2.2.1</VersionPrefix>
```

Comment:
>2 [](start = 21, length = 1)

IMO we should start being more [SemVer]([https://semver.org/)-friendly](https://semver.org/)-friendly). This is a backward-compatible feature improvement: a minor version change to `2.3.0`. It's not just a backward-compatible bug fix.

Reply:
Sure - fixed.

Reply:
I continue to be concerned about this. We are versioning many packages in sync. Diligent SemVer versioning across all these packages will greatly accelerate increase in versioning. Also, we're flat out lying about the need to update components. i.e., we are communicating that there is a breaking change everywhere, when that is not the case.

I would prefer to stick with our current model (which isn't strict SemVer, definitely admit that) unless/until we have that more complete conversation (i.e., agree that we have consensus we will uniformly apply the standard to all our various distinct packages).


---
In reply to: [390015584](https://github.com/microsoft/sarif-sdk/pull/1807#discussion_r390015584) [](ancestors = 390015584)

Reply:
Just a detail: 2.2.1 => 2.3.0 does not communicate that there is a breaking change. It says that there is enhanced, backward-compatible functionality. But perhaps you weren't referring to the particular version I was suggesting for this PR.

---
In reply to: [390504146](https://github.com/microsoft/sarif-sdk/pull/1807#discussion_r390504146) [](ancestors = 390504146,390015584)

Reply:
Given that our team approach to versioning isn't settled yet, I'll go back to 2.2.1 for this release per Mike and am happy to participate in a thread or in person discussion about how we want to handle versioning.